packages/neuron-wallet/package.json

```diff
@@ -23,19 +23,38 @@
     "rebuild:nativemodules": "electron-builder install-app-deps"
   },
   "jest": {
-    "runner": "@jest-runner/electron",
-    "testEnvironment": "@jest-runner/electron/environment",
-    "transform": {
-      "^.+\\.ts?$": "ts-jest"
-    },
-    "moduleFileExtensions": [
-      "ts",
-      "js",
-      "json",
-      "node"
-    ],
-    "testRegex": "(/tests/.*.(electron))\\.(ts?|js?)$",
-    "setupTestFrameworkScriptFile": "<rootDir>/setupTests.ts"
+    "projects": [
+      {
+        "displayName": "electron",
+        "runner": "@jest-runner/electron",
+        "testEnvironment": "@jest-runner/electron/environment",
+        "testRegex": "(/tests/.*.(electron))\\.(ts?|js?)$",
+        "transform": {
+          "^.+\\.ts?$": "ts-jest"
+        },
+        "moduleFileExtensions": [
+          "ts",
+          "js",
+          "json",
+          "node"
+        ],
+        "setupTestFrameworkScriptFile": "<rootDir>/setupTests.ts"
+      },
+      {
+        "displayName": "node",
```

Comment:
Why does the neuron wallet have two projects

Reply:
“electron” project need electron environment."node" project doesn`t

Reply:
But they cover the same test files

Reply:
different Regex."electron" is *.test.electron.ts,"node" is “.test.ts”